CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ version directory, and then changes are introduced.
 ## [v3.2.5] WIP
 
 ### Changed
+- Update hyperkube 1.10.1.
```

Comment:
Should this be `Updated` for consistency? Everything else is `Changed`, `Disabled`

Reply:
Actually past tense is inconsistent.

Reply:
Do we say hyperkube or do we just default to kubernetes, cause hyperkube might be confusing (it's an implementation detail) or we mention both?

Reply:
This is k8scloudconfig. Not exposed to the customer. I think implementation details are ok here